liquid_node/commands.py

```diff
@@ -294,6 +299,44 @@ def purge(force=False):
         print('No collections will be purged')
 
 
+def importfromdocker(path, method='link'):
```

Comment:
Let's move all of this logic to `import_from_docker.py`, it's conceptually isolated.

Reply:
It's a command; there are different commands for different functionality already in `commands.py`.  In my opinion it should stay here.

Reply:
Then move most of the implementation in the other module, there's no reason to import 5 functions instead of just one.

Reply:
The number of imported functions is irrelevant. `deploy` also uses a many functions, and some of them are in `commands.py`, although they should not be there.

Reply:
Yep, deploy should probably be its own module. So let's not clutter `commands.py` any more than we have to.

Reply:
My opinion is that `importfromdockersetup`, as a command, is in the right module. There are other functions that indeed should not be there, but it's not this function that clutters the space.

Reply:
Then keep the function and move all the logic to a new function in `import_from_docker`?